web/src/components/legend/Co2LegendRounded.tsx

```diff
@@ -0,0 +1,56 @@
+import { extent } from 'd3-array';
+import { ScaleLinear, scaleLinear } from 'd3-scale';
+import { TFunction } from 'i18next';
+import { CarbonUnits } from 'utils/units';
+
+import { spreadOverDomain } from './utils';
+
+function Co2LegendRounded({
```

Comment:
Even though this is similar to ColorBar I felt like the design was too different, so it was easier to create a new component.

Reply:
Is the ColorBar component the one used in the bottom right corner as a legend?
<img width="257" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/149581095/1383e113-3ac7-4be6-926b-c7b5673b6b39">

Maybe we can use the new component you made everywhere for consistency 🙌 It looks great!

Reply:
Yes it is. I will try to use the new component in the bottom right legend!

Reply:
It will create a bunch of merge conflicts for @VIKTORVAV99 in this PR #6961. I'm not sure if it's best that I do this first or if @VIKTORVAV99 should use the new component in his work when this is merged?

Reply:
But it could look like this. Do you like it @Alportan ?
<img width="263" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/95029552/a5f81ff6-3d4f-459d-8f6e-5076557d5bf3">

Reply:
Just go ahead and use the same components, I’ll take care of the merge conflicts in the other PR.